src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/App.config

```diff
@@ -151,6 +154,15 @@
                     </baseAddresses>
                 </host>
             </service>
+            <service name="WcfService.WcfDuplexService">
+                <endpoint address="tcp-nosecurity-callback" binding="netTcpBinding" bindingConfiguration="nettcpBindingCallback"
+                          name="nettcp-defaultBinding" contract="WcfService.IWcfDuplexService" />
+                <host>
+                    <baseAddresses>
+                        <add baseAddress="net.tcp://*:810/WindowsCommunicationFoundation" />
```

Comment:
In the latest patch, there should be no need to set up a new baseAddress. I had merged it back into the same endpoint for simplicity (i.e., `*:809`).